docs/00-lightweight-architecture-decisions.md

```diff
@@ -0,0 +1,15 @@
+# To Adopt the Lightweight Architectural Design Decision Format
+
+## Context
+
+This project is growing to the size and maturity where documenting the decisions made in it's creation will be strongly beneficial.
+
+The convention of lightweight architectural decisions [is set out by think relevance](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions). It is set out in the form of an architectural decision document, and should be read as forming the basis of this document.
```

Comment:
Let's call this an architectural decision `record` rather than `document` and see if we can squeeze in the acronym `ADR` somewhere to aid search.